### ISSUE TITLE
A way to visualize commits that only exists on a given branch

### ISSUE THREAD

**Issue by m00ndark** [ForkIssues/TrackerWin#1066]
It would be nice with a filter that would essentially show the result of `git log ^<base/default-branch> <branch>` (i.e. all commits that are reachable from `<branch>` but not from `<base/default-branch>`) in the commit tree view.

This would help visualize branch specific commits in a commit tree where the base branch has been merged into the target branch making it difficult to follow.

**Comment by DanPristupov** [comment]
Fork grays out unreachable commits for the active branch:

<img width="457" alt="Screenshot 2020-12-21 at 11 41 35" src="https://user-images.githubusercontent.com/618115/102768485-8c61e480-4381-11eb-82da-9110d5ebe7ba.png">

**Comment by m00ndark** [comment]
Right, and when filtering on a branch I don't even have to see the gray commits. While that's a great feature, it's not really what I am looking for.

I think I am looking for some kind of special view where I would be able to see _only_  those commits which are reachable from a from a specific branch (A) and _not_ reachable from another specific branch (B) (which could be the default branch by default). That would e.g. be giving me a view where I would not have see everything coming in to the branch (A) from base branch (B) merges, kind of like if I filter on the branch (A) and manually close all commit bullets with merges from the base branch (B).
